resources/i18n/hi_HI/fdmprinter.def.json.po

```diff
@@ -0,0 +1,8786 @@
+
+
+#: fdmprinter.def.json
+msgctxt "machine_feeder_wheel_diameter label"
+msgid "Feeder Wheel Diameter"
+msgstr "फीडर व्हील व्यास"
+
+
+#: fdmprinter.def.json
+msgctxt "machine_feeder_wheel_diameter description"
+msgid "The diameter of the wheel that drives the material in the feeder."
+msgstr "फीडर में मटेरियल को चलाने वाले पहिये का व्यास।"
+
+
+#: fdmprinter.def.json
+msgctxt "resolution label"
+msgid "Quality"
+msgstr "गुणवत्ता"
+
+
+#: fdmprinter.def.json
+msgctxt "resolution description"
+msgid "All settings that influence the resolution of the print. These settings have a large impact on the quality (and print time)"
+msgstr "सभी सेटिंग्स जो प्रिंट के रिज़ॉल्यूशन को प्रभावित करती हैं। इन सेटिंग्स का गुणवत्ता (और प्रिंट समय) पर बड़ा प्रभाव पड़ता है"
+
+
+#: fdmprinter.def.json
+msgctxt "layer_height label"
+msgid "Layer Height"
+msgstr "लेयर की ऊँचाई"
+
+
+#: fdmprinter.def.json
+msgctxt "layer_height description"
+msgid "The height of each layer in mm. Higher values produce faster prints in lower resolution, lower values produce slower prints in higher resolution."
+msgstr "प्रत्येक लेयर की ऊंचाई मिमी में. उच्च मान कम रिज़ॉल्यूशन में तेज़ प्रिंट उत्पन्न करते हैं, कम मान उच्च रिज़ॉल्यूशन में धीमे प्रिंट उत्पन्न करते हैं।"
+
+
+#: fdmprinter.def.json
+msgctxt "layer_height_0 label"
+msgid "Initial Layer Height"
+msgstr "प्रारंभिक लेयर की ऊंचाई"
+
+
+#: fdmprinter.def.json
+msgctxt "layer_height_0 description"
+msgid "The height of the initial layer in mm. A thicker initial layer makes adhesion to the build plate easier."
+msgstr "मिमी में प्रारंभिक परत की ऊंचाई। एक मोटी प्रारंभिक परत बिल्ड प्लेट में आसंजन को आसान बनाती है।"
+
+
+#: fdmprinter.def.json
+msgctxt "line_width label"
+msgid "Line Width"
+msgstr "लाइन चौड़ाई"
+
+
+#: fdmprinter.def.json
+msgctxt "line_width description"
+msgid "Width of a single line. Generally, the width of each line should correspond to the width of the nozzle. However, slightly reducing this value could produce better prints."
+msgstr "एक लाइन की चौड़ाई। आम तौर पर, प्रत्येक पंक्ति की चौड़ाई नोजल की चौड़ाई के अनुरूप होनी चाहिए। हालांकि, इस मान को थोड़ा कम करने से बेहतर प्रिंट तैयार हो सकते हैं।"
+
+
+#: fdmprinter.def.json
+msgctxt "wall_line_width label"
+msgid "Wall Line Width"
+msgstr "वॉल लाइन की चौड़ाई"
+
+
+#: fdmprinter.def.json
+msgctxt "wall_line_width description"
+msgid "Width of a single wall line."
+msgstr "सिंगल वॉल लाइन की चौड़ाई।"
+
+
+#: fdmprinter.def.json
+msgctxt "wall_line_width_0 label"
+msgid "Outer Wall Line Width"
+msgstr "बाहरी दीवार की लाइन की चौड़ाई"
+
+
+#: fdmprinter.def.json
+msgctxt "wall_line_width_0 description"
+msgid "Width of the outermost wall line. By lowering this value, higher levels of detail can be printed."
+msgstr "सबसे बाहरी दीवार लाइन की चौड़ाई। इस मान को कम करके, उच्च स्तर के विवरण मुद्रित किए जा सकते हैं।"
+
+
+#: fdmprinter.def.json
+msgctxt "wall_line_width_x label"
+msgid "Inner Wall(s) Line Width"
+msgstr "भीतरी दीवार (ओं) लाइन की चौड़ाई"
```

Comment:
The "(s)" is to indicate that there is either a single wall or multiple walls. The letter "S" shouldn't be translated litterer here